subsys/bluetooth/mesh/Kconfig

```diff
@@ -895,6 +895,49 @@ config BT_MESH_BLOB_REPORT_TIMEOUT
 	help
 	  The timer value that Pull BLOB Transfer Server uses to report missed chunks.
 
+config BT_MESH_RX_BLOB_CHUNK_SIZE
+	int "BLOB Server chunk size"
+	default 0
+	range 0 377 if BT_MESH_RX_SEG_MAX = 32
```

Comment:
Note: kconfig does not permit arithmetics, and using https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/kconfig/kconfigfunctions.py#L922 is also not an option, as BT_MESH_RX_SEG_MAX is not set at time of such a function call.